src/rprblender/properties/render.py

```diff
@@ -615,19 +615,12 @@ def init_rpr_context(self, rpr_context, is_final_engine=True, use_gl_interop=Fal
             rpr_context.set_parameter(pyrpr.CONTEXT_TEXTURE_CACHE_PATH, self.texture_cache_dir)
 
         if isinstance(rpr_context, (context.RPRContext2, RPRContextHybridPro)):
-            # check for custom ocio setting and use it if exists
-            ocio_config_path = os.path.join(utils.package_root_dir(),'ocio_config',
-                                            utils.BLENDER_VERSION, 'config.ocio')
-
-            if not os.path.exists(ocio_config_path):
-                # set ocio config file to blender included one
-                ocio_config_path = os.path.join(bpy.utils.resource_path('LOCAL'),
-                                                'datafiles', 'colormanagement',
-                                                'config.ocio')
-
-            rpr_context.set_parameter(pyrpr.CONTEXT_OCIO_CONFIG_PATH, ocio_config_path)
+            # set ocio config file to blender included one
+            rpr_context.set_parameter(pyrpr.CONTEXT_OCIO_CONFIG_PATH,
+                                      os.path.join(bpy.utils.resource_path('LOCAL'),
+                                                   'datafiles', 'colormanagement',
+                                                   'config.ocio'))
```

Comment:
Does blender include ocio for all versions?

Reply:
blender 3.5: OCIO 2.2.0, ocio_profile_version: 2
blender 3.4: OCIO 2.1.1, ocio_profile_version: 2
blender 3.1-3.3: OCIO 2.0.0, ocio_profile_version: 2
blender 2.93-3.0: OCIO 2.0.0, ocio_profile_version: 1